src/test/groovy/com/athaydes/automaton/SwingBaseForTests.groovy

```diff
@@ -312,6 +314,37 @@ abstract class SwingDriverWithSelectorsTest extends SimpleSwingDriverTest {
 	}
 
 	@Test
+	void testMoveTo_TableCell_CustomRenderer() {
+		def tModel = [
+				[ firstCol: new SimpleStringProperty('1 - 1'), secCol: new SimpleStringProperty('1 - 2') ],
+		]
+		def cellRenderer = new DefaultTableCellRenderer(){
+			@Override
+			protected void setValue(Object value) {
+				setText(((SimpleStringProperty) value).getValue())
```

Comment:
Looks like you copied this from Java, but as you know, this in Groovy should be much nicer:

```groovy
text = value.value
```